exercises/concept/welcome-to-tech-palace/.docs/introduction.md

```diff
@@ -11,19 +11,30 @@ firstName := "Jane"
 Strings can be concatenated via the `+` operator:
 
 ```go
-fullName := "Jane" + " " + "Austen"
+fullName := "Jane" + " " + "Austen" // Output: "Jane Austen"
```

Comment:
Recently we've been discussing how using "Output" on statements that are not print statements can be confusing. We are looking to remove these instances. Here you can keep using variables but remove the "Output" bit from the comments. Alternatively, you should keep the "Output" in the comments, and use print statements for the strings. Because of the examples with special characters, maybe using a print statement would be the best solution here.

For more context about this change, see our previous discussion in #2156